src/modules/createlistelement.js

```diff
@@ -0,0 +1,15 @@
+const listElement = {
+  description: '',
+  completed: false,
+  index: 0,
+};
+
+const createlistelement = (str, arraylist) => {
+  const newlistelement = Object.create(listElement);
+  newlistelement.description = str;
+  newlistelement.index = arraylist.length + 1;
+  newlistelement.completed = false;
```

Comment:
You can try to erase this line because you are adding already the default value for `completed` set to `false` above. 👍

Reply:
I tried deleting it and now it doesn't have the completed attribute shown in the local storage, so it appears that objects don't have a default attribute.